db/migrate/20220706180729_createpost.rb

```diff
@@ -0,0 +1,11 @@
+class Createpost < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+
+      t.timestamps
+  end
+end
```

Comment:
- [ ] You are missing the association between the user and the post table It can be done here with 
`t.references :author, foreign_key: { to_table: 'users' }` # this will create author_id and a corresponding index  or you can also add column and index in separate commands